Library/Formula/ddar.rb

```diff
@@ -9,16 +9,22 @@ class Ddar < Formula
 
   depends_on "xmltoman" => :build
   depends_on :python if MacOS.version <= :snow_leopard
-  depends_on "protobuf" => "with-python"
+  depends_on "protobuf"
 
   def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    ENV.prepend_path "PYTHONPATH", Language::Python.homebrew_site_packages
```

Comment:
this should not be necessary; it won't help pick up protobuf because protobuf is brought in through a .pth file and pythonpath doesn't help python pick up .pths